drivers/hmis/app/graphql/types/hmis_schema/service.rb

```diff
@@ -56,7 +56,18 @@ def sub_type_provided
     end
 
     def service_type
-      load_ar_association(object, :custom_service_type)
+      custom_service_types_scope = Hmis::Hud::CustomServiceType.where(data_source_id: object.data_source_id)
+      case object.owner_type
+      when 'Hmis::Hud::Service'
+        # ideally we'd lazy load this
+        custom_service_types_scope.to_a.detect do |cst|
```

Comment:
because we load all custom service types here, the query is the same so rails query cache should mean we aren't actually hitting the db. However this could be made more efficient with a custom data loader